extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -225,9 +225,14 @@ export class AmpAnalytics extends AMP.BaseElement {
         this.instrumentation_.createAnalyticsGroup(this.element);
 
     if (this.config_['transport'] && this.config_['transport']['iframe']) {
+      const ampAdResourceId = user().assert(
```

Comment:
if you do assertString I believe it will save you the type cast below

Reply:
Done.